### DISCUSSION TITLE
NOS 2.8.7 full missing ses.asm68k

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#67] [General]
/SES.ASM68K.
ASM68K NOT FOUND
/

I was kinda expecting an error abort...just not this one...

**Comment by wschaub** [comment]
SES is a twisted maze of procedures and it's looking for the procedure by that name, or a a file inside the SES catalog where the m68k assembler lives.  The SES we have doesn't have those tools. but there is a tape here that has a copy of SES that can be used to build the CDCNET sources. that should have the m68k version of CYBIL and possibly an assembler too (I've not done much with it, I just know it exists)  https://cdc.sjzoppi.com/cfarchive:content:cdcnet_sources

**Reply by KC5VDJ** [reply]
That site looks like it has a lot, but any attempt to register for access....  ![Screenshot_20230813-145256.png](https://github.com/kej715/DtCyber/assets/136194499/a5a6faa0-7646-4a6c-8c74-5a0b82abd478)

Been trying for some months....

**Reply by jim02762** [reply]
Content is being migrated over to retro1.org. The new home for the mail archive is https://archer.retro1.org .

**Comment by pa0gri** [comment]
I user it quite often to measure DtCyber speed.

But I see I still am unkown to the archive on sjzoppi.com (was gvdg on the old “thing”.)

I had the SES tools uploaded looong time ago on controlfreaks.

Gerard.

**Reply by KC5VDJ** [reply]
When is he going to fix his registration page?  Been trying for months.

![Screenshot_20230813-145256.png](https://github.com/kej715/DtCyber/assets/136194499/7804ab33-29da-4b51-b41a-35b5c540a11c)

**Reply by drboone** [reply]
Maybe just send https://github.com/sz10101 him a note and ask?

**Reply by KC5VDJ** [reply]
Apparently he has been busy migrating everything to retro1.org.  Kevin posted current info.

**Comment by kej715** [comment]
An error exit -does- occur. For example, if you create the following CCL
procedure:

.PROC,P.

SES.ASM68K

NOTE./SUCCESS

EXIT.

NOTE./FAILURE


When this procedure is executed, "FAILURE" is printed, indicating that an
"error abort" -does- occur.

cheers,
Kevin
